.github/workflows/cd.yml

```diff
@@ -24,28 +52,44 @@ jobs:
         env:
           release_token: ${{ secrets.FINETUNER_RELEASE_TOKEN }}
 
-  create-release:
-    name: Create release
-    needs: update-doc
+#  create-release:
+#    name: Create GitHub release
+#    needs: update-docs
```

Comment:
Why is this commented?

Reply:
it needs additional stuff to work. I left it commented out for now